### PR TITLE
Allen/swap req util

### PR DESCRIPTION
**Pull request by algtan**
Added a utility function that creates a swap request. Also created unit testing for it.

### CODE REVIEW

File: src/utils/swapQuotes.ts [hunk 1]
```diff
@@ -26,15 +43,11 @@ export async function swapQuotes(
...
-    toWallet: currencyWallets[1],
-    fromCurrencyCode: currencyWallets[0].currencyInfo.currencyCode,
-    toCurrencyCode: currencyWallets[1].currencyInfo.currencyCode,
-    nativeAmount: '1000000',
-    quoteFor: 'from' as const
-  }
+  const swapRequest = createFromSwapRequest(
+    currencyWallets[0],
+    currencyWallets[1],
+    1000000
```
Comment (LPeterson18) [inline src/utils/swapQuotes.ts:49]: I would recommend using a constant for this number (maybe for the 0 and 1 as well).  As it is, it is a little unclear what the number means or represents.

**Comment by LPeterson18** [comment]
I only had one comment. Reassigning to Allen

File: src/utils/swapQuotes.ts [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
 import {
   EdgeAccount,
   EdgeCurrencyWallet,
   EdgePluginMap,
-  EdgeSwapQuote
+  EdgeSwapQuote,
+  EdgeSwapRequest
 } from 'edge-core-js'
 
+export const createFromSwapRequest = (
```
Comment (thehobbit85) [inline src/utils/swapQuotes.ts:10]: Instead of "regular" function this should be a cleaner.
I'll show you tomorrow what I mean.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comment

**Comment by LPeterson18** [comment]
Changes look good to me.

**Pull request merged**